specifications/README.md

```diff
@@ -6,7 +6,10 @@ keywords: Devicetree, Specification, release, v1.0
 ---
 ## **The Devicetree Specification**
 
-<a href="/downloads/devicetree-specification-v0.1-20160524.pdf" target="_blank"><strong>Device Tree Specification Release 0.1</strong></a><br>
```

Comment:
We probably want to keep the prior versions.

Reply:
How do you want it listed on the page ? Just beneath 0.2 ?

Reply:
Yes. Perhaps have "Current Release" and "Past Releases" sections.

The other option would be to have separate pages for current and past.